### PR TITLE
embedding analysis: knn probing

### PR DESCRIPTION
**Pull request by JochenSiegWork**
add experiments to analyze embeddings using k-nearest neighbor probing, which is common in representation learning and useful for read across in regulatory toxicology.

### CODE REVIEW

**Comment by akshatzalte** [comment]
Update:
I was able to reproduce all the results.
This means that having the validation set is generally harmful for the analysis. The low validation loss doesn't translate to better kNN probing results due to inclusion of validation fingerprints in the kNN training set. I will take a closer look at what is happening here. Any guidance from @JochenSiegWork and team would be helpful! 
In the meantime, I propose removing the validation set and simply adding the CheMeleon (frozen) fingerprint to the analysis with the aesthetic changes.
